src/components/index.ts

```diff
@@ -1,6 +1,10 @@
 export { default as Card } from './Card/Card';
 export { default as UniversalTable } from './UniversalTable/UniversalTable';
+<<<<<<< HEAD
```

Comment:
I guess some merge went badly wrong here...

Reply:
Clened up

Reply:
if you consider it's done, then resolve a conversation please.

Reply:
extra rows cleared
![image](https://github.com/user-attachments/assets/8f01e316-b56f-4d3c-925f-c4d8a9f1228a)